src/Uno.UWP/UI/ViewManagement/UISettings.cs

```diff
@@ -1,27 +1,74 @@
 #pragma warning disable 108 // new keyword hiding
 #pragma warning disable 114 // new keyword hiding
+using System.Collections.Concurrent;
 using Uno;
+using Uno.Helpers.Theming;
+using Windows.Foundation;
+using Windows.UI.Core;
 
 namespace Windows.UI.ViewManagement
 {
-	[NotImplemented]
-	public  partial class UISettings 
-	{
-		[NotImplemented]
-		public bool AnimationsEnabled
+	/// <summary>
+	/// Contains a set of common app user interface settings and operations.
+	/// </summary>
+	/// <remarks>Events on this class are fired as long as the instance is alive.
+	/// To ensure the class does not get garbage collected, keep a strong reference to it.</remarks>
+	public partial class UISettings 
+	{		
+		private static readonly ConcurrentDictionary<WeakReference<UISettings>, object> _instances = new ConcurrentDictionary<WeakReference<UISettings>, object>();
+		private readonly WeakReference<UISettings> _weakReference;
+
+		public UISettings()
+		{			
+			_weakReference = new WeakReference<UISettings>(this);
+			_instances.TryAdd(_weakReference, null);
+		}
+
+		~UISettings()
 		{
-			get
+			_instances.TryRemove(_weakReference, out var _);
+		}
+
+		internal static void OnColorValuesChanged()
+		{			
+			foreach (var instance in _instances)
 			{
-				global::Windows.Foundation.Metadata.ApiInformation.TryRaiseNotImplemented("Windows.UI.ViewManagement.UISettings", "AnimationsEnabled");
-#if __WASM__
-				// Animations are marked as not supported for wasm until implemented properly.
-				return false;
-#else
-				return true;
-#endif
+				var weakReference = instance.Key;
+				if (weakReference.TryGetTarget(out var uiSettings))
+				{
+					uiSettings.ColorValuesChanged?.Invoke(uiSettings, null);
+				}
 			}
 		}
 
+		public event TypedEventHandler<UISettings, object> ColorValuesChanged;
+
+#if !__ANDROID__
+		public bool AnimationsEnabled => true;
+#endif
+
+		public Color GetColorValue(UIColorType desiredColor)
+		{
+			var systemTheme = SystemThemeHelper.SystemTheme;
+			return desiredColor switch
+			{
+				UIColorType.Background =>
+					systemTheme == SystemTheme.Light ? Colors.White : Colors.Black,
+				UIColorType.Foreground =>
+					systemTheme == SystemTheme.Light ? Colors.Black : Colors.White,
+				// The accent color values match SystemResources.xaml in Uno.UI
+				// as we can't access Application resources from here directly.
+				UIColorType.Accent => Color.FromArgb(255, 51, 153, 255),
```

Comment:
This brings us closer to supporting `SystemAccentColor` properly :) https://docs.microsoft.com/en-us/windows/uwp/design/style/color#accent-color

Reply:
I actually started on Accent color implementation for Android, but there are some catches:
- The accent color can potentially be anything (its "rules" are not clearly defined)
- It will require adding some color helpers for RGB -> HSL conversion, so the color brightness can be changed properly to scale to darker/lighter

Reply:
Added as #4444 :-)